src/runtime/HalideRuntimeHexagonHost.h

```diff
@@ -122,15 +121,28 @@ extern int halide_hexagon_set_performance_mode(void *user_context, halide_hexago
 extern int halide_hexagon_set_performance(void *user_context, halide_hexagon_power_t *perf);
 // @}
 
-/** Set the default priority for Halide Hexagon user threads:
+// Used with FASTRPC_THREAD_PARAMS req ID
+struct remote_rpc_thread_params {
+    int domain;      // Remote subsystem domain ID, pass -1 to set params for all domains
+    int prio;        // user thread priority (1 to 255), pass -1 to use default
+    int stack_size;  // user thread stack size, pass -1 to use default
+};
+
+/** Set the default priority/stack_size for Halide Hexagon user threads:
```

Comment:
Does set_remote_thread_params apply to all FastRPC threads, or just the calling thread? At first I thought it was the latter, but now the docs make it sound like the former...

Reply:
I'm not sure about this. Will confirm with Pranav and get back to you.

Reply:
It will apply to all fastrpc threads spawned by the process and not just he calling thread.